src/js/rx/components/Prescription.jsx

```diff
@@ -22,7 +22,7 @@ class Prescription extends React.Component {
     const content = this.props.prescriptions.items.find((rx) => {
       return rx.id === refillId;
     });
-    this.props.dispatch(openRefillModal(content.attributes));
+    this.props.openRefillModal(content.attributes);
```

Comment:
While we're back on Prescription.jsx, I'd like to see the refill handler passed along to this component as a prop so that it can be a proper component rather than a container.

Outside the scope of these changes though; maybe I'll pick it up when I have some time to refactor. Might be able to get around doing the `Array.find` to find the corresponding item in the store as well.